bin/miden-cli/Cargo.toml

```
@@ -37,7 +35,7 @@ uuid = { version = "1.10", features = ["serde", "v4"] }
 clap = { version = "4.5", features = ["derive"] }
 comfy-table = { version = "7.1" }
 figment = { version = "0.10", features = ["toml", "env"] }
-miden-client = { version = "0.10", path = "../../crates/rust-client", features = ["sqlite", "tonic"] }
+miden-client = { version = "0.10", path = "../../crates/rust-client", features = ["std", "sqlite", "tonic"] }
```

This should probably be `concurrent` instead:
```suggestion
miden-client = { version = "0.10", path = "../../crates/rust-client", features = ["concurrent", "sqlite", "tonic"] }
```